mdpow/workflows/dihedrals.py

```diff
@@ -31,13 +31,25 @@
 .. autofunction:: plot_violins
 
 """
-
+#^need to update function list
 import os
 import pathlib
 import numpy as np
 import pandas as pd
 
+# add to dependencies for install and tests
+import cairosvg
+import svgutils
+import svgutils.compose as sc
+import svgutils.transform as sg
```

Comment:
Do you use `sc` and `sg` that often to make it worthwhile abbreviating it? The code is probably clearer when you just use `svgutils.transform`. 

Also, is `compose` still used? In any case, `svgutils.compose` is also clearer.

I'd prefer the full versions.